docs/site.json

```diff
@@ -1,7 +1,7 @@
 {
   "baseUrl": "",
-  "titlePrefix": "DevelopersCo.",
-  "titleSuffix": "AddressBook Level-3",
+  "titlePrefix": "CS2103 Team-12",
```

Comment:
Should we make it T12-1? I think the "T" in T12 stands for Thursday, 12 being our tutorial time and our actual group being group 1 in that slot